### ISSUE TITLE
Text() widget not rendered after app was relaunched in background and brought to front

### ISSUE THREAD

**Issue by derzhavets** [flutter/flutter#55208]
My app (iOS) recording 10 hours of bluetooth data being in background. Therefore if it's killed to reclaim memory, iOS relaunches it in background.

When the app relaunched in background I see this error (a lot):

> Execution of the command buffer was aborted due to an error during execution. <private>

but everything continues to work fine (recording data). However if I bring my app to front, Text() widgets either not rendered or rendered partly (see image). 

I tried to manually rebuild all the widgets - didn't help.

--------------------------

Normal launch:
![IMG_0030](https://user-images.githubusercontent.com/18610129/79768646-bfa42d80-8333-11ea-9b51-ebfff668ec8d.PNG)

Bringing to front after the app was relaunched in background (notice that alert dialog is ok):
![IMG_0031](https://user-images.githubusercontent.com/18610129/79768685-c92d9580-8333-11ea-929f-beaf1ab2e327.PNG)
![IMG_0032](https://user-images.githubusercontent.com/18610129/79768710-d054a380-8333-11ea-8124-c907c7b2489b.PNG)
![IMG_0033](https://user-images.githubusercontent.com/18610129/79768713-d2b6fd80-8333-11ea-985c-62011efd2993.PNG)

----------------------------
There is really nothing special in my widget structure, but here it is anyway (unfolded for simplicity):
```
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        centerTitle: true,
        title: InteractiveTitle(title: title),
        elevation: 0,
        actions: <Widget>[ServiceCounter(), AppbarConnectionIndicators()],
        flexibleSpace: AppBarDecoration(),
      ),
      body: Column(
        crossAxisAlignment: CrossAxisAlignment.stretch,
        children: <Widget>[
          Flexible(
            flex: 7,
            child: Container(
              decoration: BoxDecoration(
                color: Color.fromARGB(255, 35, 35, 35),
                image: DecorationImage(
                  image: AssetImage('assets/animation_recording.gif'),
                  fit: BoxFit.fitHeight,
                ),
              ),
            ),
          ),
          Flexible(
            flex: 3,
            child: Container(
              decoration: BoxDecoration(color: Color.fromARGB(255, 35, 35, 35)),
              child: Column(
                mainAxisAlignment: MainAxisAlignment.spaceBetween,
                children: <Widget>[
                  Spacer(),
                  Container(
                    margin: EdgeInsets.only(bottom: 10.0),
                    child: Text(
                      S.of(context).recordingTitle,
                      style: TextStyle(
                        fontSize: Theme.of(context).textTheme.headline6.fontSize,
                        color: Colors.white,
                      ),
                    ),
                  ),
                  Container(
                      margin: EdgeInsets.only(bottom: 10.0),
                      child: StreamBuilder(
                        stream: sl<TestingManager>().elapsedTimeStream,
                        initialData: 0,
                        builder: (BuildContext context, AsyncSnapshot<int> snapshot) {
                          if (snapshot.hasData) {
                            return Text(
                              '${S.of(context).elapsed_time}: ${TimeUtils.convertSecondsToHMmSs(snapshot.data)}',
                              style: TextStyle(
                                fontSize: Theme.of(context).textTheme.headline6.fontSize,
                                color: Colors.white.withOpacity(0.9),
                              ),
                            );
                          } else {
                            return Text("");
                          }
                        },
                      )),
                  Spacer(),
                  ButtonsBlock(
                    moreActionButton: null,
                    nextActionButton: ButtonModel(
                      action: () {
                        if (sl<SystemStateManager>().testDataAmountState == TestDataAmountState.MINIMUM_PASSED) {
                          _confirmEndTest(context);
                        } else {
                          _showTestTimerDialog(context);
                        }
                      },
                      text: S.of(context).btnEndRecording,
                    ),
                  ),
                  Spacer(flex: 2)
                ],
              ),
            ),
          )
        ],
      ),
      backgroundColor: backgroundColor,
      resizeToAvoidBottomPadding: false,
    );
  }
```

**Comment by VladyslavBondarenko** [comment]
Hi @derzhavets 
could you provide your `flutter doctor -v`? 
and with what device are you experiencing this?

**Comment by derzhavets** [comment]
Hi @VladyslavBondarenko 

I am using iPhone 8 with iOS 13.4

Here is flutter doctor output:
```
Michaels-Mini:~ derzhavets$ flutter doctor -v
[✓] Flutter (Channel master, v1.16.4-pre.69, on Mac OS X 10.15.4 19E287, locale
    en-IL)
    • Flutter version 1.16.4-pre.69 at /Users/derzhavets/flutter
    • Framework revision 19e7db585d (3 weeks ago), 2020-04-01 11:16:01 -0700
    • Engine revision c9506cb8e9
    • Dart version 2.8.0 (build 2.8.0-dev.18.0 eea9717938)

 
[!] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/derzhavets/Library/Android/sdk
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor
      --android-licenses

[✓] Xcode - develop for iOS and macOS (Xcode 11.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.4.1, Build version 11E503a
    • CocoaPods version 1.9.1

[!] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)

[✓] IntelliJ IDEA Community Edition (version 2019.3.4)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 44.0.3
    • Dart plugin version 193.6911.31

[!] VS Code (version 1.38.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    ✗ Flutter extension not installed; install from
      https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[✓] Connected device (1 available)
    • Michael Derzhavets’s iPhone • ef167d41f226a124160be3587820ff6b00ea1773 •
      ios • iOS 13.4

! Doctor found issues in 3 categories.
Michaels-Mini:~ derzhavets$ 

```

**Comment by derzhavets** [comment]
Hi, it was supposed that my issue will be fixed by [this fix](https://github.com/flutter/flutter/issues/55215), but it didn't.

Here is my flutter doctor:

> [✓] Flutter (Channel master, v1.18.0-9.0.pre.14, on Mac OS X 10.15.4 19E287, locale en-IL)
>     • Flutter version 1.18.0-9.0.pre.14 at /Users/derzhavets/flutter
>     • Framework revision dd88b20435 (29 hours ago), 2020-04-25 19:02:03 -0700
>     • Engine revision 2b94311a77
>     • Dart version 2.9.0 (build 2.9.0-3.0.dev 94178e920e)

Can I provide any additional input to help to fix the issue?

Could it be that running 'flutter upgrade' wasn't enough to pick up engine changes?

Looking forward for a resolution! I got app release blocked by this issue :(

**Comment by TahaTesser** [comment]
Hi @derzhavets 
Can you please steps to reproduce the issue
Can you please provide your `flutter doctor -v` your `flutter run --verbose` and a complete reproducible minimal code sample (including code for background task) 
Thank you

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
